components/codeflare/codeflare.go

```diff
@@ -35,6 +35,18 @@ type CodeFlare struct {
 	components.Component `json:""`
 }
 
+func (c *CodeFlare) Init(_ cluster.Platform) error {
+	var imageParamMap = map[string]string{
+		"codeflare-operator-controller-image": "RELATED_IMAGE_ODH_CODEFLARE_OPERATOR_IMAGE", // no need mcad, embedded in cfo
+	}
+
+	if err := deploy.ApplyParams(ParamsPath, imageParamMap); err != nil {
```

Comment:
can we also update docs on https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/pkg/deploy/envParams.go#L67
just to be clear that it only update if it does not match otherwise it wont do anything

Reply:
yeah, leftover.  (It's a bit counter intuitive that access nil map does not crash)

Reply:
You mean 450e19fd1afa ("deploy: ApplyParams: do not write params.env if there are no updates (#1186)") functionality?

Reply:
yes, by reading the comments, i do not feel it reflects what the fuction is doing with recent chagnes

Reply:
It's anyway unrelated to this patchset, can come as fixup for the aforementioned patch.